tools/sigma/sigmac.py

```diff
@@ -201,6 +207,7 @@ def main():
         sys.exit(0)
     elif len(cmdargs.inputs) == 0:
         print("Nothing to do!")
+        print(deprecation_warning_message)
```

Comment:
As it is only when some one use wihtout rule , there is no need for the "Please stop contributing backends to this tool" part.

Reply:
I think it's good to have the message regarding contribution present here and add a link to pySigma and sigma-cli, such that people who want to use our contribute directly are pointed to the right direction.

Reply:
OK , let go for this.